scss/detail.scss

```diff
@@ -115,17 +128,29 @@ $break-medium: 1024px;
   }
 
   .book-image {
-    align-self: center;
-    margin: 0 0 21px 0;
+    text-align: center;
+    margin: 0 0 6px 0;
```

Comment:
margin-bottom

Reply:
@manueltuero I use this property for reseat margin when the screen use a media query